controllers/iamrole_controller.go

```diff
@@ -107,76 +113,97 @@ var roleTrust = `{
 
 //HandleReconcile function handles all the reconcile
 func (r *IamroleReconciler) HandleReconcile(ctx context.Context, iamRole *iammanagerv1alpha1.Iamrole) (ctrl.Result, error) {
-	log := r.Log.WithValues("id", iamRole.ObjectMeta.UID, "iamrole", fmt.Sprintf("k8s-%s", iamRole.ObjectMeta.Namespace))
-
+	log := r.Log.WithValues("request_id", ctx.Value(requestId), "iamrole", iamRole.Name)
+	log.Info("And the state is ", "state", iamRole.Status.State)
 	role, _ := json.Marshal(iamRole.Spec.PolicyDocument)
-	fmt.Println(string(role))
 	roleName := fmt.Sprintf("k8s-%s", iamRole.ObjectMeta.Namespace)
 	input := awsapi.IAMRoleRequest{
 		Name:             roleName,
-		PolicyName:       fmt.Sprintf("k8s-%s-policy", iamRole.ObjectMeta.Namespace),
-		Description:      "bla bla",
+		PolicyName:       "custom",
```

Comment:
I think we should create an issue to manage all constants in one place. I might become difficult as the code grows.

Reply:
Agreed. Lets make sure we handle this in https://github.com/keikoproj/iam-manager/issues/18 along with properties